### ISSUE TITLE
Organização dos arquivos

### ISSUE THREAD

**Issue by migmoroni** [cfbcursos/Novo-Javascript#1]
Um aspecto comum no processo de crescimento de um projeto é este começar com um padrão de organização. Todavia todo padrão tem limite. Quando ultrapassou a aula 100, percebo o problema de essas aulas ficarem em posições "erradas", assim prejudicando na visualização destas no repositório.

Gosto de em certos momentos organizar arquivos e aqui, como estou realizando seu curso, senti vontade em lhe ajudar em organizar as aulas, em um novo padrão. Sei que esse tipo de ação gasta um tempo que entendo o porque não ter sido realizado até então.

Dessa forma, antes de me dar o trabalho de fazer isso e nem saber se aceitaria essa mudança, venho ver com você Bruno, se está aberto a receber esta ajuda, assim vendo qual padrão seguimos nesta mudança ou prefere deixar desta maneira, por alguma razão.

**Comment by flameastro** [comment]
Concordo, nessa parte acho que o Bruno foi um pouco longe demais em colocar todo o projeto em uma pasta só
